includes/Modules/AdSense.php

```diff
@@ -537,10 +537,16 @@ protected function create_adsense_earning_data_request( array $args = array() )
 			$opt_params['maxResults'] = (int) $args['limit'];
 		}
 
-		$host = wp_parse_url( $this->context->get_reference_site_url(), PHP_URL_HOST );
-		if ( ! empty( $host ) ) {
-			$opt_params['filter'] = 'DOMAIN_NAME==' . $host;
-		}
+		$hostnames = array_unique(
+			array_map(
+				function ( $site_url ) {
+					return 'DOMAIN_NAME==' . wp_parse_url( $site_url, PHP_URL_HOST );
+				},
+				$this->permute_site_url( $this->context->get_reference_site_url() )
+			)
+		);
```

Comment:
Just as previous comment – the permuted site URLs will always be unique so no need for this extra call 👍 
```suggestion
		$hostnames = array_map(
			function ( $site_url ) {
				return 'DOMAIN_NAME==' . wp_parse_url( $site_url, PHP_URL_HOST );
			},
			$this->permute_site_url( $this->context->get_reference_site_url() )
		);
```

Reply:
Oops! Sorry about that, you were right 👍  🤦

Reply:
🙂👍